third_party/BUILD

```diff
@@ -605,6 +605,39 @@ filegroup(
     ],
 )
 
+pkg_tar(
+    name = "cpp_util",
+    srcs = [
+        "//src/main/cpp/util:embedded_java_tools",
+    ],
+    extension = "tar.gz",
+    # Permissions -rwxr-xr-x
```

Comment:
nit: These permission comments are unnecessary, that's what the documentation of pkg_tar is for in order to avoid repeating this comment again and again.

Reply:
Done.